### PR TITLE
FIX: Make sure ``--cifti-output`` requires at least one of ``fsaverage{5,6}``

### PR DESCRIPTION
**Pull request by oesteban**
Fixes poldracklab/niworkflows#309

### CODE REVIEW

File: fmriprep/cli/run.py [hunk 1]
```diff
@@ -573,7 +573,19 @@ def build_workflow(opts, retval):
...
+    # Reduce to unique space identifiers
+    output_spaces = sorted(set(opts.output_space))
+
+    # If FS is not run, drop all fs* output spaces
+    if not opts.run_reconall:
+        output_spaces = [item for item in output_spaces if not item.startswith('fs')]
+
+    # Map fsaverage to fsaverage6
+    if 'fsaverage' in output_spaces:
+        logger.warning('Selected "fsaverage" output space has been mapped to "fsaverage6"')
```
Comment (effigies) [inline fmriprep/cli/run.py]: I don't think we should override user selected output spaces.

Comment (oesteban) [reply fmriprep/cli/run.py]: fsaverage is an alias for fsaverage6, isn't that correct?

Comment (oesteban) [reply fmriprep/cli/run.py]: The reason for this is that our cifti interface fails if it is passed "fsaverage". We could do the mapping silently as FS does, but I prefer being explicit.

Comment (effigies) [reply fmriprep/cli/run.py]: No. fsaverage6 is a deformed order 6 icosahedron. fsaverage is order 7.

Comment (effigies) [reply fmriprep/cli/run.py]: The reason we require fsaverage6 for cifti is because fsaverage would be huge.

Comment (oesteban) [reply fmriprep/cli/run.py]: Gotcha

**Comment by oesteban** [comment]
Sorry for the history rewrite, I forgot that ``--no-fs-reconall`` and ``--cifti-output`` are mutually exclusive. After all that mess, the corresponding tests are here: https://circleci.com/workflow-run/c6b97b23-e35c-4d63-a9a7-eb7807299c7f

**Pull request merged**